molgenis/molgenis-emx2
# feat: enable custom prefixes/triples for RDF API

What are the main changes you did:
- Prefixes are now user-defined (per database).
- Addition of custom triples (note that these will be added to EVERY API call for that database)
- Default is no prefixes except namespace of database itself (can be changed by adding `settings: _settings/rdf_api` to templates).
  - Alternative is adding an extra advanced setting in `_SYSTEM_` for global prefixes/RDF.
- If invalid RDF is added, settings will be stored but API will fail.

In case multiple database are called, the prefixes/custom triples well be merged together.

how to test:
- Create a new database of "FAIR_DATA_HUB" and change the values in the advanced setting "custom_rdf". Any changes should immediately be present in the API.

todo:
